backstop/scenarios/footer.js

```diff
@@ -1,13 +1,32 @@
-const { scenario } = require('../scenarioHelper');
+const { scenario, hover, focus } = require('../scenarioHelper');
 
 module.exports = [
-  scenario('footer hidden below viewport', {
+  // Footer types
+  scenario('Type hidden below viewport', {
     selectors: ['viewport'],
     viewports: [{ width: 800, height: 600 }],
   }),
-  scenario('footer visible in viewport', {
+  scenario('Type visible in viewport', {
     selectors: ['viewport'],
     viewports: [{ width: 800, height: 600 }],
     scrollToSelector: ['.iui-legal-footer'],
   }),
+
+  // Hover states
+  //// Footer link
+  scenario('State hover', {
+    selectors: ['viewport'],
+    viewports: [{ width: 800, height: 600 }],
+    scrollToSelector: ['.iui-legal-footer'],
+    actions: [hover('#test-link')],
+  }),
+
+  // Focus states
+  //// Footer link
+  scenario('State focus', {
```

Comment:
Is it possible to have both focus and hover in the same test? 🤔

Reply:
Indeed it is. I think we should do that to reduce the number of test images.
![image](https://user-images.githubusercontent.com/9084735/115445872-6be95000-a1e4-11eb-8201-850af40dc491.png)

Reply:
I think we thought about this before.

Reply:
Oooh, I love this suggestion.  I've pushed the update.